lib/fog/aws/parsers/compute/describe_images.rb

```diff
@@ -66,9 +66,9 @@ def end_element(name)
               end
             else
               case name
-              when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType'
+              when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType','creationDate'
```

Comment:
[`Time.parse` should do it for you](https://github.com/fog/fog-aws/blob/09a405cc1474636f436406fc31c7b8baebaead8f/lib/fog/aws/parsers/cloud_formation/list_stacks.rb#L21)